updatecli/updatecli.d/nodejs.yml

```diff
@@ -0,0 +1,91 @@
+---
+title: "Bump Node.js version"
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ requiredEnv .github.owner }}"
+      repository: "{{ requiredEnv .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  latestVersion:
+    kind: githubRelease
+    name: "Get the latest Node.js version"
+    spec:
+      owner: "nodejs"
+      repository: "node"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      versionFilter:
+        kind: regex
+        pattern: '*LTS*'
```

Comment:
Please help me to check the `versionFilter`.

Reply:
Maybe it is wrong. Node.js LTS release versions have no features.

Reply:
I'm adding a suggestion. Please keep in mind that the "versionFilter" only works with the releases IDs, not the title. It means that you have to fix the version to 16.x as it is the current LTS.
The "semver" filter would be the best choice (see suggestion).